lib/bundler/shared_helpers.rb

```diff
@@ -203,6 +203,17 @@ def md5_available?
 
   private
 
+    def validate_bundle_path
+      return unless Bundler.bundle_path.to_s.match(File::PATH_SEPARATOR)
+      message = "WARNING: Your bundle path contains a '#{File::PATH_SEPARATOR}', " \
```

Comment:
Remove 'WARNING:` as this is an error that will exit Bundler. We use warning when we need to notify the user about something wrong but Bundler can still continue with the action.